jaxrs-spec/src/main/asciidoc/boilerplate.adoc

```diff
@@ -16,24 +16,20 @@
 
 == Boilerplate
 
-WARNING: This document contains just boilerplate information. It explicitly is *not* a complete specification. In particular, it misses all normative text. To learn about compliance with this specification, refer to "JAX-RS: Java™ API for RESTful Web Services" (Version 2.1, Final Release July 13, 2017).
+WARNING: This document contains just boilerplate information. It explicitly is *not* a complete specification.
```

Comment:
Why do you think this spec in particular needs this warning but none of the other specs do?

Reply:
I do not think that actually. Actually I do think that *all* Boilerplate Specs should countain that warning so readers understand that *essential* information is missing.

Reply:
Although I understand what @mkarg is saying, looking at other boilerplate specs (e.g., JSONB and JSONP), they don't include any of this information. In the interest of time, we should follow the same approach and continue this discussion at a later time.